benjr70/Smart-Smoker-V2
# feature/SS2-34-init-backend

init backend

just used the next cli to init a new project named backend

also added an exampled notifications endpoint on how to add new ones

testing:

1. in terminal navigate to  /backend
2. run `npm i`
3. run `npm run start`
4. go to `http://localhost:3000/`
5. can also see the swgger UI at `http://localhost:300/api`

should see a helloworld
